rita.rst

```diff
@@ -42,12 +36,12 @@ Run the installer:
 
 ::
 
-   sudo bash ./install.sh
-   
+   sudo bash ./install.sh --disable-bro
+
 Configuration
 -------------
 
-You must set your own values for ``InternalSubnets`` in ``/etc/rita/config.yaml`` before importing
+You should set your own values for ``InternalSubnets`` in ``/etc/rita/config.yaml`` before importing
```

Comment:
With 3.2.0 we have RFC1918 networks enabled by default. We still recommend users verify these settings match their own situation but it's no longer required.